tst/EnergyPlus/unit/CurveManager.unit.cc

```diff
@@ -476,6 +476,48 @@ TEST_F(EnergyPlusFixture, DivisorNormalizationDivisorOnly)
+                                                      "Dimensionless,                         !- Output Unit Type",
+                                                      ",                                      !- External File Name",
+                                                      ",                                      !- External File Column Number",
+                                                      ",                                      !- External File Starting Row Number",
+                                                      "2.0,                                   !- Value 1",
+                                                      "4.0;                                   !- Value 2",
+
+                                                      "Table:IndependentVariableList,",
+                                                      "y_values_list,                         !- Name",
+                                                      "x_values_1;                            !- Independent Variable Name 1",
+
+                                                      "Table:IndependentVariable,",
+                                                      "x_values_1,                            !- Name",
+                                                      ",                                      !- Interpolation Method",
+                                                      ",                                      !- Extrapolation Method",
+                                                      ",                                      !- Minimum value",
+                                                      ",                                      !- Maximum value",
+                                                      ",                                      !- Normalization Reference Value",
+                                                      "Dimensionless                          !- Output Unit Type",
+                                                      ",                                      !- External File Name",
+                                                      ",                                      !- External File Column Number",
+                                                      ",                                      !- External File Starting Row Number",
+                                                      ",",
+                                                      "2.0,                                   !- Value 1",
+                                                      "3.0;                                   !- Value 3"});
+
+    ASSERT_TRUE(process_idf(idf_objects));
+    EXPECT_EQ(0, state->dataCurveManager->NumCurves);
+
+    EXPECT_THROW(Curve::GetCurveInput(*state), std::runtime_error);
+    EXPECT_TRUE(this->err_stream->str().find("Normalization divisor entered as zero") != std::string::npos);
+}
```

Comment:
New unit test that sets up a simple table lookup with zero valued normalization divisor.  It first makes sure the runtime error is thrown during curve get input, and then also makes sure there is a specific mention of the normalization divisor in the error output.  Note I'm not comparing the whole error output because I don't care about overall output message structure, I just want to make sure the divisor error is, in fact, emitted.